### PR TITLE
Docopt export descriptor#392

### PR DESCRIPTION
**Pull request by DarrinFong**
# Change
 _This is a 'partial' implementation and requires #524 to complete_

```bosh import dcpt [OUTPUT] [DOCOPT_PY_INPUT]```

Creates a valid boutiques descriptor from a valid docopt script inside of a .py file.

### CODE REVIEW

File: tools/python/boutiques/templates/template_docopt_descriptor.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (gkiar) [inline tools/python/boutiques/templates/template_docopt_descriptor.json]: As discussed, potentially use `bosh create` with no arguments to generate a template from which you can build?

File: tools/python/boutiques/schema/examples/docopt_to_argparse/sample_pattern.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (gkiar) [inline tools/python/boutiques/schema/examples/docopt_to_argparse/sample_pattern.json]: Unless we use these in tests (to verify the "expected" interpretation of command args), we should remove the `pattern` files from the repo

**Comment by DarrinFong** [comment]
Flaky tests
https://travis-ci.org/DarrinFong/boutiques/builds/568490831

**Comment by DarrinFong** [comment]
> I'm happy to merge - is it ready, or are you building any more in?

Cool! It's ready

**Comment by gkiar** [comment]
Great! Nicely done :)

**Pull request merged**